securing_apps/topics/overview/supported-platforms.adoc

```diff
@@ -1,5 +1,7 @@
 === Supported Platforms
 
+For platforms, {project_name} supports both OpenID Connect and SAML.
```

Comment:
Perhaps: "{project_name} enables you to protect applications running on different platforms and using different technology stacks using OpenID Connect and SAML protocols." Or similar ...